### ISSUE TITLE
Readonly field value max-width too long in form layout

### ISSUE THREAD

**Issue by yardensachs** [unfoldadmin/django-unfold#786]
**What version of Unfold are you using?**
0.39.0

**Did you checked changelog/commit history, if the bug is not already fixed?**
Not fixed

**Did you searched other issues, if the bug is not already fixed?**
Not fixed

**Describe your issue**
When using a field readonly, and having it not on a fieldset row, Unfold will use `max-w-4xl`, but to be aligned with the rest of the layout, it should be `max-w-2xl`.

This happens in this file:
https://github.com/unfoldadmin/django-unfold/blob/46863a55c67841924cbca0892a5f1e87c2033420/src/unfold/templates/unfold/helpers/field_readonly_value.html#L1C22-L1C31

**Current:**
![SCR-20241006-hvwc](https://github.com/user-attachments/assets/0035d813-07df-4151-8b96-6d187bf1ce52)

**after fix:**
![SCR-20241006-hxcs](https://github.com/user-attachments/assets/7af16ae0-899d-4be8-81f3-bf2b098f7f58)

**Comment by yardensachs** [comment]
https://github.com/unfoldadmin/django-unfold/blob/46863a55c67841924cbca0892a5f1e87c2033420/src/unfold/templates/unfold/helpers/field_readonly_value.html#L1C22-L1C31

```diff
- <div class="readonly max-w-4xl ...
+ <div class="readonly max-w-2xl ...
```

**Comment by lukasvinclav** [comment]
Thanks for report. Fixed in #802.
